src/libstatistics_collector/moving_average_statistics/types.cpp

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <cstdint>
```

Comment:
Hm, I don't see why this is needed here.  In `rolling` and `iron`, we need this because we reference `uint64_t`, but in `humble` that is not the case.

Can you please explain more about the error you are seeing?

Reply:
![image](https://github.com/ros-tooling/libstatistics_collector/assets/37519090/e8168091-e3f4-4a0d-a5ee-06653a46656f)

After just following the guide stated here: https://docs.ros.org/en/humble/Installation/Alternatives/RHEL-Development-Setup.html#rhel-dev-get-ros2-code

And only just following the guide, the error does pop for me. 
In git it also says i am on the branch Humble, so it does seem to be the case that it gets used from what I see. 
Am I incorrect in my understanding?

Reply:
![image](https://github.com/ros-tooling/libstatistics_collector/assets/37519090/8e9b6393-2a08-4089-8b38-49b56bd888c8)
And if I check types.hpp in the humble branch we do reference `uint64_t`

Reply:
@Relmnicoder If we are using the `uint64_t` in the `types.hpp`, it looks like it would be reasonable to add `#include <cstdint>` there.

Reply:
I'm seeing this issue on fedora40

Reply:
Yes, that's true, `uint64_t` is indeed referenced in types.hpp.  But as far as I can tell, it is already included there: https://github.com/ros-tooling/libstatistics_collector/blob/humble/include/libstatistics_collector/moving_average_statistics/types.hpp#L19

For those that this is failing for, can you please do a `git rev-parse HEAD` and paste the output?